### ISSUE TITLE
Apply Bold to coded text flagged as Important

### ISSUE THREAD

**Issue by ccbogel** [ccbogel/QualCoder#474]
Text codings flagged as important, currently have a notation to such, in a mouse hover tooltip.
Make these more prominent by applying Bold.

Note - general text annotations are also marked Bold.

**Comment by AndrzejWawa** [comment]
I have suggestion:
1) make important codings bold,
2) however, at the same time, add underline to annotations (except of bold).

Underline the annotations will make them more visible than only bold, as well as will distinguish them from important codings.

**Comment by ccbogel** [comment]
I have made them bold.
But underline will not work well. for example if you have dark mode and a dark text on a light coded section. if there is an annotation inside this text, the underline will also be dark, and not visible.
Closing for now.

**Comment by glocalglocal** [comment]
Now if there is an important fragment in a text, then all codings are displayed in bold. If you add a new coding, it will also appear in bold.

**Comment by AndrzejWawa** [comment]
Ok, now, with the annotation information, it is ok. Yes, I noted this problem glocal indicated, however, it occurs for me only in the case of files, where you previously applied the important mark function. In other words:
1) in the files, where I never used the important flag, everything is fine, codings font is normal, when you mark importance, they became bold, when you remove, they become normal.
2) but, in the files, where I used important flag, every codings is normal, disregard if they are marked as important or not. Also all new codings are bold. For me it is nor rather a problem, because I used important flag in only one file.
